app/models/group.rb

```diff
@@ -604,4 +604,12 @@ def limit_inheritance
   def self.with_one_coordinator
     published.select{ |g| g.admins.count == 1 }
   end
+
+  def default_cover_photo
+    if self.default_group_cover
+      /^.*(?=\?)/.match(self.default_group_cover.cover_photo.url).to_s
```

Comment:
It is, because paperclip takes the id of the object into account to look for the object. So, given 
```
Group.new cover_photo_file_name: 'test.jpg', id: 15
```

it'll go and look in the Fog store for
```
groups/cover_photos/000/000/15/test.jpg
```
(or something like that), which it won't find because we haven't uploaded it.

Next step to fix that would be to override the paperclip default path to something which doesn't involve the id, which is really scary to me and I think would require a data migration of all of our existing groups. :/